### PR TITLE
config-tools: refine misc_cfg.h.xsl and lib.xsl

### PR DESCRIPTION
**Pull request by yuchuyang**
The whitespaces between delimar saperated elemnts of a list causes
acrn:get-common-clos-max and acrn:find-list-min returns unexpected
numbers. Add translate to remove the whitspaces.

Initialize the minimum value to an extreme large value to avoid the
return value of acrn:find-list-min is empty.

Tracked-On: #6515
Signed-off-by: Yang,Yu-chu <yu-chu.yang@intel.com>

### CODE REVIEW

File: misc/config_tools/xforms/misc_cfg.h.xsl [hunk 1]
```diff
@@ -145,8 +145,8 @@
 <!-- MAX_CACHE_CLOS_NUM_ENTRIES:
   Max number of MBA delay entries corresponding to each CLOS. -->
 <xsl:template name="rdt">
-  <xsl:variable name="rdt_resource" select="normalize-space(substring-before(substring-after(//CLOS_INFO, 'rdt resources supported:'), 'rdt resource clos max:'))" />
-  <xsl:variable name="rdt_res_clos_max" select="normalize-space(substring-before(substring-after(//CLOS_INFO, 'rdt resource clos max:'), 'rdt resource mask max:'))" />
+  <xsl:variable name="rdt_resource" select="translate(normalize-space(substring-before(substring-after(//CLOS_INFO, 'rdt resources supported:'), 'rdt resource clos max:')), ' ', '')" />
```
Comment (dbkinder) [inline misc/config_tools/xforms/misc_cfg.h.xsl:148]: Do you also want to watch for tabs or carriage returns as a potential white space?

**Comment by wenlingz** [comment]
@NanlinXie please help approve this PR, thanks

**Comment by yuchuyang** [comment]
Please hold this PR.

**Comment by yuchuyang** [comment]
Closed for some discussion.

**Pull request closed without merging**